ci/docs.go

```diff
@@ -27,6 +27,29 @@ pagination_prev: null
 # CLI Reference
 `
 
+// Build the docs website
+func (d Docs) Site() *Directory {
+	return dag.
+		Docusaurus(
+			d.Dagger.Source,
+			DocusaurusOpts{Dir: "/src/docs", DisableCache: true},
+		).
+		Build()
+}
+
+// Build the docs server
+func (d Docs) Server() *Container {
+	nginxConfig := dag.CurrentModule().Source().File("docs-nginx.conf")
```

Comment:
:eyes: this file isn't part of this PR:

```
marshal: json: error calling MarshalJSON for type *dagger.Container: input: blob
.withDirectory.withNewFile.withNewFile.withNewFile.directory.file resolve: lstat
 /ci/docs-nginx.conf: no such file or directory
```

Reply:
> 👀 this file isn't part of this PR:

Thanks, fixed!

> The docusaurus module also looks to have a Serve field directly as well.

That function runs what I consider to be a dev version `npm run serve`. I chose to serve the static files with nginx. This more closely matches the current production environment. Next step will be to apply the same redirect configuration (currently expressed in a proprietary netlify.tom file).